src/SandboxSizeParameters.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Sandboxes\Api;
+
+class SandboxSizeParameters
+{
+    private ?int $storageSize_GB;
+
+    public function __construct(?int $storageSize_GB = null)
```

Comment:
do we need an argument here?  why not just use the static methods or the setters?

Reply:
we can keep it, I can just imagine myself looking at this in ta month and thinking wtf is 128? :-D then checking the constructor.

it's nbd

Reply:
ok, that's a very good point actually.. I'll update it